### DISCUSSION TITLE
Weird circle position on pausing game

### DISCUSSION THREAD

**Discussion by BennyDioxide** [ppy/osu#22933] [Q&A]
### Type

Game behaviour

### Bug description

Just normaly open a map and play, but when Workrave, a program that assists me to take a rest assists, then I pause the game and the thing happened.

### Screenshots or videos

![Screenshot_20230319_112020](https://user-images.githubusercontent.com/80139133/226151793-8eaa5981-543a-40e9-8131-1b69bc200c69.png)

### Version

2023.305.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/11010132/database.log)
[input.log](https://github.com/ppy/osu/files/11010133/input.log)
[network.log](https://github.com/ppy/osu/files/11010134/network.log)
[performance.log](https://github.com/ppy/osu/files/11010135/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11010136/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11010137/updater.log)

**Comment by bdach** [comment]
I'm not sure what I'm supposed to be looking at on the screenshot. What "circle" are you referring to, here?

**Comment by Axilotl17** [comment]
I think they mean how you must click your cursor after pausing to resume the map, but I can't find it either.

**Comment by peppy** [comment]
Please try and reproduce and attach a video.
